src/store/modules/order/getters.ts

```diff
@@ -3,8 +3,8 @@ import OrderState from "./OrderState";
 import RootState from "../../RootState";
 
 const getters: GetterTree<OrderState, RootState> = {
-  getOrder(state) {
-    return state.list;
+  getPurchaseOrders(state) {
+    return state.purchaseOrders;
```

Comment:
We should return a deep clone. If we return a reference, there could be possibility that it is mutated across the code without the state flow